src/pandas_profiling/report/structure/report.py

```diff
@@ -276,23 +402,26 @@ def get_report_structure(config: Settings, summary: dict) -> Root:
                 )
             )
 
-        sample_items = get_sample_items(summary["sample"])
+        sample_items = get_sample_items(config, summary["sample"])
         if len(sample_items) > 0:
             section_items.append(
                 Container(
                     items=sample_items,
-                    sequence_type="list",
+                    sequence_type="tabs",
                     name="Sample",
                     anchor_id="sample",
                 )
             )
 
-        duplicate_items = get_duplicates_items(summary["duplicates"])
+        duplicate_items = get_duplicates_items(config, summary["duplicates"])
         if len(duplicate_items) > 0:
             section_items.append(
                 Container(
                     items=duplicate_items,
-                    sequence_type="list",
+                    sequence_type="batch_grid",
+                    batch_size=len(duplicate_items)
+                    if isinstance(duplicate_items, list)
+                    else 1,
```

Comment:
Smells like this should be parenthesized for ease of reading (but really it would be better if `duplicate_items` just always was a list)

Reply:
This check was outdated, `duplicate_items` always is a list